jaas-login/src/main/java/com/wipro/ats/bdre/jaas/login/JAASLoginModule.java

```diff
@@ -34,13 +34,13 @@
  */
 public class JAASLoginModule implements LoginModule {
 
-    private static Logger LOGGER = Logger.getLogger(JAASLoginModule.class);
+    private static final Logger LOGGER = Logger.getLogger(JAASLoginModule.class);
 
     // initial state
     private Subject subject;
     private CallbackHandler callbackHandler;
-    private Map sharedState;
-    private Map options;
+    private Map<String, ?> sharedState;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "sharedState" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1068)